### ISSUE TITLE
[Carbon 10: Notification] Inline notification visual issues

### ISSUE THREAD

**Issue by elizabethsjudd** [carbon-design-system/carbon#2294]
## Detailed description

> Describe in detail the issue you're having.

1. The warning notification has a transparent background so if it's applied to any thing other than a white background the color doesn't stay consistent (could use the sass `mix` function to blend it with white to make it opaque if it's not in the IBM DL color palette)
2. Timestamp styling is not supported in inline notifications but is in toast notifications.  I'd like to apply the same styles for both notification types without having to copy and past Carbon's code in to my style sheets.

> Is this issue related to a specific component?

Notification

> What did you expect to happen? What happened instead? What would you like to see changed?

1. The background on the information notification is opaque
2. I don't have to copy and paste Carbon's code to style a timestamp

> What browser are you working in?

Chrome

> What version of the Carbon Design System are you using?

v10.0.0

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

Watson Health Pattern & Asset Library (PAL)

## Steps to reproduce the issue

Issue 1
1. Change the page background color when using a warning notification

Issue 2
1. Add a time stamp to an inline notification

## Additional information

<img width="645" alt="Screen Shot 2019-04-11 at 5 08 53 PM" src="https://user-images.githubusercontent.com/1275779/55993208-88109900-5c7c-11e9-802a-aefb5e6ba58f.png">

**Comment by elizabethsjudd** [comment]
Also found an issue where the title of the notification doesn't wrap to multiple lines if it is longer than the width of the notification (used Carbon's code pen in screen shot)

<img width="661" alt="Screen Shot 2019-04-12 at 4 47 43 PM" src="https://user-images.githubusercontent.com/1275779/56065446-d8522e80-5d42-11e9-87b4-90917ccf3eba.png">

**Comment by elizabethsjudd** [comment]
Our designer also noticed the close button does not have a hover state

**Comment by rauhala** [comment]
Opaque background in warning notification is an issue also with toast notifications. Please see attached image which displays the problem in more detail.

<img width="484" alt="Screenshot 2019-04-28 at 18 09 55" src="https://user-images.githubusercontent.com/16488270/56866371-01c8b680-69e1-11e9-86e1-b0db82638415.png">

**Comment by dakahn** [comment]
@carbon-design-system/design can you comment on the opaque background here?

**Comment by aagonzales** [comment]
Notification styles are being updated: https://github.com/carbon-design-system/carbon-components/issues/2354

**Comment by elizabethsjudd** [comment]
@aagonzales is the update for #2354 also going to resolve the other issues mentioned with the overflow and timestamp stylings? It looks like #2367 is meant to resolve this issue but I don't see any updates for these mentioned issues.

**Comment by aagonzales** [comment]
The inline notifications were not originally designed to have a time stamp. We can maybe add it as a optional feature for inline but I'd like to see some research and guidelines around when and where a timestamp should be used with notifications before making it part of the default. If you have any information or thoughts around this topic we'd love to hear it.

**Comment by tw15egan** [comment]
Still needs to be fixed, #2746 did not solve this

https://codepen.io/tjegan/pen/NVgRVg

<img width="879" alt="Screen Shot 2019-05-17 at 9 23 48 AM" src="https://user-images.githubusercontent.com/11928039/57942194-b5f18a80-7885-11e9-9879-1df691d9dc86.png">

**Comment by elizabethsjudd** [comment]
@janedepgen

Can you add feedback as to why we have timestamps in our inline notifications at WH or if we should be updating ours to align closer to Carbon's guidelines? https://github.com/carbon-design-system/carbon/issues/2294#issuecomment-489736536

**Comment by janedepgen** [comment]
@elizabethsjudd we should update to align to Carbon - thinking about the use case of inline notifications vs toast notifications, it makes more sense to just include timestamps for the toast variant 👍

**Comment by shixiedesign** [comment]
Just checked, long text is no longer an issue in both vanilla and react. Looks like we are good with no time-stamp on inline too.

**Comment by elizabethsjudd** [comment]
@carbon-design-system/developers has this been released?  We have pulled in v10.3.1 and still have the long title issues with our notifications

<img width="1552" alt="Screen Shot 2019-06-21 at 2 32 23 PM" src="https://user-images.githubusercontent.com/1275779/59943889-6b899d80-9431-11e9-8d91-b52f3a33beff.png">

**Comment by elizabethsjudd** [comment]
@carbon-design-system/developers 
In v10.3.2 this is still an issue.  Please re-open this issue and you can assign it to me.

current v10.3.2 implementation
<img width="712" alt="Screen Shot 2019-07-03 at 10 41 52 AM" src="https://user-images.githubusercontent.com/1275779/60601097-83411a00-9d7f-11e9-9fdd-855e6a6c39a5.png">

remove "whitespace: nowrap" from title
(I tested a short title and removing the optional subtitle as well and it worked correctly)
<img width="712" alt="Screen Shot 2019-07-03 at 10 42 07 AM" src="https://user-images.githubusercontent.com/1275779/60601098-83d9b080-9d7f-11e9-8651-11193b101685.png">
